src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String drawFigure();
```

Comment:
```suggestion
    String draw();
```
Why drawFigure? Tomorrow we will create `class Car implements Drawable ` - it has no connection to Figure. Remember OOP Abstraction principle. Drawable - is behavior. Do not mix it with implementation.